packages/google_sign_in/google_sign_in_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.6.2
+
+* Update functions without a prototype to avoid deprecation warning.
```

Comment:
```suggestion
* Updates functions without a prototype to avoid deprecation warning.
```

I always get scolded for this one 🙂 
https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
> Entries should use present tense indicative for verbs, with "this version" as an implied subject. For example, "Adds cool new feature.", not "Add" or "Added".